include/SDL_video.h

```diff
@@ -158,6 +158,7 @@ typedef enum
                                          system or user changing the window size. */
     SDL_WINDOWEVENT_MINIMIZED,      /**< Window has been minimized */
     SDL_WINDOWEVENT_MAXIMIZED,      /**< Window has been maximized */
+    SDL_WINDOWEVENT_FULLSCREEN,     /**< Window has been made fullscreen */
```

Comment:
Thanks for adding this event!

We should move it to the end, so that it is the last event - that will help preserve [ABI compatibility](https://en.wikipedia.org/wiki/Application_binary_interface) by not changing the numeric values of existing events.

Reply:
Fixed in the latest commit! Thanks!